tasks/cockpit-tasks-webhook.yaml

```diff
@@ -5,20 +5,20 @@ items:
 - kind: ReplicationController
   apiVersion: v1
   metadata:
-    name: release
+    name: tasks-webhook
```

Comment:
How about just naming that `webhook`? It should be an implementation detail of the RC, service, and pod that this is running in the tasks container.

Reply:
true